python/ray/experimental/workflow/step_function.py

```diff
@@ -64,11 +68,13 @@ def options(self,
                 If it's set to be true, (Optional[R], Optional[E]) will be
                 returned.
                 If it's false, the normal result will be returned.
+            name(str): The name of this step which can be used to fetch
```

Comment:
```suggestion
            name(str): The name of this step, which will be used to generate the
               step_id of the step. The name will be used directly as the step id if
               possible, otherwise deduplicated by appending _N suffixes.
```